db/migrate/20230921055904_alter_table_users.rb

```diff
@@ -0,0 +1,19 @@
+class AlterTableUsers < ActiveRecord::Migration[7.0]
+  def up
+    add_column :users, :role, :integer, default: 0, null: false
+    add_column :users, :designation, :string
+    add_column :users, :email, :string
+    add_column :users, :password_digest, :string
+    add_column :users, :password_confirmation, :string
+    rename_column :users, :fist_name, :first_name
```

Comment:
Since we have model level validations for some of the fields, let's also add the db validations for null:false for the corresponding fields